.circleci/config.yml

```diff
@@ -167,3 +147,4 @@ workflows:
               only:
                 - raft-tdp-main
                 - raft-review
+                - raft-staging
```

Comment:
In general I think there's a desire among the dev team to allow deploys from any branch. Since that would introduce more complexity, holding on that for now and planning to break it into an incremental improvement to follow — specifically for the staging site, but should probably apply to both.